### ISSUE TITLE
Blurred images after data augmentation

### ISSUE THREAD

**Issue by pransen** [Megvii-BaseDetection/YOLOX#565]
I am trying to fine-tune the model on my own dataset.

However, while looking into the images (for initial few iterations) after augmentation has been applied, I find the images to be quite blurry. Is this expected ?

**Comment by Joker316701882** [comment]
@pransen Yes. It's normal. That's why we close the augmentation for the last 15 epochs.

**Comment by pransen** [comment]
Is it also normal for the images in the mosaic to overlap ? It's as if the images are blended together similar to cv2.addWeighted().

**Comment by Joker316701882** [comment]
@pransen The overlap you mentioned is caused by Mixup. It is also closed for the last 15 epochs.
